hugo/data/home/home.yml

```diff
@@ -31,6 +29,9 @@ projects:
 
 roadmap:
   display: true
+  content:
```

Comment:
Since there would need to be styling involved, I would wait for the asset and then do the change.